### PR TITLE
Create Edit Spend Venue (FRONT) > 584

### PR DESCRIPTION
**Pull request by thejoefriel**
#### Issues this PR relates to
#584

I've created all bits except state handling and client side validation of the images which I think makes more sense to do when connecting the back

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

**Comment by fadeomar** [comment]
- [ ] I think you forgot to add the `add a spend venues` button 
Screenshot from 2020-07-09 13-41-12

  - [ ] Also the `edit venue` button is no longer disabled, in page `/admin/spend-venue/4/details`
Screenshot from 2020-07-09 13-43-13

  - [ ] The image is not the same as the figma and the wireframes do not have a link to `view spend activity` 
image

  - [ ] all input fields in wireframe take 4 columns on the grid, not 6 in desktop view 
https://github.com/yalla-coop/tempo/blob/f2d98571e3691c5d676b82b838180160c6719dc7/client/src/pages/SpendVenueForm/index.js#L246

  - [ ] when open edit spend venue  the title of listing section still `Create Spend Venue Listing`
 not `Edit Spend Venue Listing`
 as in the wireframe 
https://github.com/yalla-coop/tempo/blob/f2d98571e3691c5d676b82b838180160c6719dc7/client/src/pages/SpendVenueForm/index.js#L291

  - [ ] too much horizontal spacing between the checkbox and It's label, you can just remove the left padding 
image

File: client/src/pages/SpendVenueForm/AccountSettings.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          name="accountLastName"
+          value={accountLastName}
+          setValue={handleInput}
+          error={errors.accountLastName}
+          justify="center"
+        />
+      </Col>
+    </Row>
+    <Row mb={edit ? 9 : 5}>
+      <Col w={[4, 6, 6]}>
```
Comment (fadeomar) [inline client/src/pages/SpendVenueForm/AccountSettings.js:61]:
```suggestion
      <Col w={[4, 4, 4]}>
```

Comment (thejoefriel) [reply client/src/pages/SpendVenueForm/AccountSettings.js:61]: I've commented on this below. I think 6 columns is better. 4 looks a bit too narrow unless you have it fully full screen

**Comment by thejoefriel** [comment]
Hey @fadeomar , thanks have made the changes and pushed back up. Two things though:
- I think actually seeing it in web format that 6 col is better than 4 col. So I'd like to leave as is and run by Tempo on that if that's ok? Is simple enough to switch back if they're not happy 
- I don't get the Create Spend Venue issue, just checking you have latest version?

**Pull request merged**